RELEASE-NOTES.txt

```diff
@@ -9,6 +9,7 @@
 * The Site → Blog Posts → View WebView will no longer show any Calypso UI
 * Fixed an issue with Twitter publicize connections
 * Fixed an issue where the site icon wasn't updating correctly after switching to another site
+* Fixes an issue which can result in creating gaps in Reader
```

Comment:
It wasn't entirely clear what "gaps" meant when I read this.  I first thought it was the margin/padding of the layout.  Users might not know what "gaps" refers to either, but if we intend to change this after review from Editorial, we can leave it as-is for now.

Reply:
That makes sense, it wouldn't hurt to improve it even before it goes to the Editorial. How about something like this:

```
* Fixes an issue which can result in missing posts in the Reader
```

Reply:
That works for me.  Users and Editorial will understand that better.

Reply:
Fixed in 200064e4332f7c237bd1cfec7a0df6ec592c0a65.